### DISCUSSION TITLE
Einrichtung HmIP-FALMOT-C12 ohne HmIP Wandthermostat

### DISCUSSION THREAD

**Discussion by stlorenz** [OpenCCU/OpenCCU#2579] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe einen Fußbodenheizungscontroller FALMOT-C12 mit 7 belegten Aktoren. 
Dieser ist über den AccessPoint direkt an die Zentrale angelernt und ich kann auch alle Kanäle sehen. 

Aktuell habe ich noch die "alten" Homematic Wandthermostate ohne IP in den Räumen, kann diese also nicht direkt an den Controller anlernen. 

In der Übersicht unter Geräte sehe ich nun die belegten Aktoren, alle stehen auf 0%. 
Ich kann diese aber weder manuell öffnen noch die einzelnen Kanäle in einem Programm ansteuern. 

Geht das? Oder brauche ich zwingend die HmIP Thermostate?

**Comment by jens-maus** [comment, marked as answer]
> Geht das? Oder brauche ich zwingend die HmIP Thermostate?

Nein, das geht nicht. Du brauchst zwingend einen HmIP Wandthermostat.

Ein Homematic bzw. homematicIP Fußbodenaktor wie der `HmIP-FALMOT-C12` braucht **immer** zwingend einen direktverknüpften Wandthermostaten (z.B. `HmIP-WTH`). Auch kann man die Fußbodenheizungskanäle (selbst wenn der `HmIP-WTH` direktverknüpft ist) nicht selbst auf/zu machen wie das hier und da manchmal von Nutzern angedacht ist. Die Fußbodenheizungssteuerung via `HmIP-FALMOT` bzw. den anderen Fußbodenheizungsaktoren der Homematic-Reihe funktioniert **immer** vollkommen autark zwischen dem entsprechenden Fußbodenheizungsaktor (hier `HmIP-FALMOT-C12`) und einem zugeordnetem Wandthermostaten (z.B. `HmIP-WTH`). D.h. die Logik wie weit ein Fußbodenheizungskreis auf/zu gemacht wird obliegt alleinig der im Fußbodenheizungsaktor bzw. Wandthermostaten internen Logik. Man kann lediglich via Einstellen des Soll-Termperatur-Wertes im Wandthermostaten Einfluß auf die Fußbodenheizungsregelung nehmen. Was anderes ist nicht vorgesehen und wird vmtl. auch niemals anders umgesetzt werden von eQ3. Ist aber auch nicht notwendig, denn die Fußbodenheizungsaktoren soll ja auch vollkommen selbstständig in Kombination mit einem Wandthermostaten alles automatisch regeln.

**Reply by stlorenz** [reply]
Danke. 
Ich sehe das auch so, die Regelung sollte vollkommen autark sein. Dann kann auch weniger kaputt gehen. 
Ich wollte die Ventile jetzt eben für den Übergang, bis die `HMIP-WTH-1` da sind, manuell fahren. 
Dann muss ich die eben erst einmal wieder umbauen.

**Reply by Panoru84** [reply]
Hallo @jens-maus,
ich möchte an dieses alte Frage nochmal anknüpfen.
Kann man auch nicht irgendwie virtuelle Wandthermostate in der CCU anlegen und diese verknüpfen?
Ich möchte als Wandthermostate gerne Sonoff NSPanell nutzen, weil die mit dem Bildschirm einfach noch mehr Möglichkeiten bieten.(Wie gut die Temperaturwerte der Panels sind wird sich zeigen).
Meine Idee war dann über IoBroker diese Raumwerte an die CCU bzw. FALMOT-C12 weiterzugeben.
Geht das irgendwie?

Übrigens vielen Dank für deine super tolle Arbeit!!! 🥳

**Reply by jens-maus** [reply]
Nein, sowas geht nicht.

**Comment by alexirion** [comment]
Ich versuche auch gerade den Ventilwert des Falmot Kanals 1 einzustellen mit:

http://192.168.178.98/addons/xmlapi/statechange.cgi?ise_id=3061&new_value=0.75

Aber der LEVEL Wert wird immer noch als "" angezeigt. Geht das denn wirklich nicht, den Ventilwert selber zu setzen? Brauche ich also wirkliuch unbedingt Thermostate, die das per Direktverknüpfung setzen?

Auszug aus http://192.168.178.98/addons/xmlapi/state.cgi?device_id=3028:

```
<channel name="HmIP-FALMOT-C12 001B9F29A2B41E:1" ise_id="3052">
<datapoint name="HmIP-RF.001B9F29A2B41E:1.DEW_POINT_ALARM" type="DEW_POINT_ALARM" ise_id="3053" value="false" valuetype="2" valueunit="" timestamp="1745422607"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.EMERGENCY_OPERATION" type="EMERGENCY_OPERATION" ise_id="3054" value="false" valuetype="2" valueunit="" timestamp="1745422607"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.EXTERNAL_CLOCK" type="EXTERNAL_CLOCK" ise_id="3058" value="false" valuetype="2" valueunit="" timestamp="1745422607"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.FROST_PROTECTION" type="FROST_PROTECTION" ise_id="3059" value="false" valuetype="2" valueunit="" timestamp="1745422607"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.HUMIDITY_LIMITER" type="HUMIDITY_LIMITER" ise_id="3060" value="false" valuetype="2" valueunit="" timestamp="1745422607"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.LEVEL" type="LEVEL" ise_id="3061" value="" valuetype="4" valueunit="100%" timestamp="1744971029"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.LEVEL_STATUS" type="LEVEL_STATUS" ise_id="3062" value="1" valuetype="16" valueunit="" timestamp="1745422515"/>
<datapoint name="HmIP-RF.001B9F29A2B41E:1.VALVE_STATE" type="VALVE_STATE" ise_id="3063" value="7" valuetype="16" valueunit="" timestamp="1745422607"/>
</channel>
```

**Reply by jens-maus** [reply]
> Brauche ich also wirkliuch unbedingt Thermostate, die das per Direktverknüpfung setzen?

Ja! Und nein, das wird sich vermutlich nie ändern.

**Reply by mawi2684** [reply]
Auch nicht mit CCU-JACK?
Ich habe gelesen man könnte den FALMOT-12 auf manuellen Modus schalten:
`device/set/<device_id>/<channel>/CONTROL_MODE → 1`
und dann mit VALVE_ADJUSTMENT arbeiten
`device/set/<device_id>/<channel>/VALVE_ADJUSTMENT → 0–100`

Müsste mal jemand ausprobieren, da ich selbst HmIP-STH nutze brauche ich das nicht;
aber wenn das tatsächlich über MQTT funktioniert, könnte das schon ein Kostenvorteil sein gegenüber den HM-IP Thermostaten, wenngleich auch über hohen Konfigurationsaufwand zu kompensieren.

**Reply by stlorenz** [reply]
Ich kann keine Aussage zur technischen Umsetzung machen, finde den Ansatz aber nicht so sinnvoll.

Durch den Wegfall der Regelung fällt auch der automatische hydraulische Abgleich weg. 

Den Kostenvorteil verheizt du am Ende nur.

**Comment by alexirion** [comment]
Hier wird aber beschrieben, dass nach einem Firmware Update der Falmot die Ventilwerte schreibbar sein sollen:

https://homematic-forum.de/forum/viewtopic.php?t=83219

Ist das nun richtig, oder nicht?
